modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling parameters using the ROSA CLI
+
+There are several parameters you can add to the cluster creation command to adjust autoscaling options. The table explains all the configurable autoscaler parameters when using the ROSA CLI (`rosa`).
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Example/Instruction
+
+|`--autoscaler-balance-similar-node-groups`
+|Identify node groups with the same instance type and label set, and aim to balance respective sizes of those node groups.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-skip-nodes-with-local-storage`
+|If set, cluster autoscaler does not delete nodes with pods that have local storage, for example, EmptyDir or HostPat.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-log-verbosity _int_`
+|Autoscaler log level. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-log-verbosity 4`
+
+|`--autoscaler-max-pod-grace-period _int_`
+|Gives pods graceful termination time before scaling down, measured in seconds. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-pod-grace-period 0`
+
+|`--autoscaler-pod-priority-threshold _int_`
+|The priority that a pod must exceed to cause the cluster autoscaler to deploy additional nodes. Replace _int_ in the command with the number you want to use, can be negative.
+|`integer`
+|`--autoscaler-pod-priority-threshold -10`
+
+|`--autoscaler-ignore-daemonsets-utilization`
+|If set, the cluster-autoscaler ignores daemon set pods when calculating resource utilization for scaling down.
+|`boolean`
+|Add it to set to true, do not add to set to false.
+
+|`--autoscaler-max-node-provision-time _string_`
+|Maximum time that the cluster-autoscaler waits for node to be provisioned. Replace _string_ in the command with an integer and time unit (ns,us,µs,ms,s,m,h).
+|`string`
+|`--autoscaler-max-node-provision-time 35m`
+
+|`--autoscaler-balancing-ignored-labels _strings_`
+|A comma-separated list of label keys that cluster autoscaler should ignore when considering node group similarity. Replace _string_ in the command with the string you want to use.
+|`string`
+|`--autoscaler-balancing-ignored-labels topology.ebs.csi.aws.com/zone,alpha.eksctl.io/instance-id`
+
+|`--autoscaler-max-nodes-total _int_`
+|Maximum amount of nodes in the cluster, including the autoscaled nodes. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-nodes-total 1000`
```

Comment:
something I just fixed also in the CLI (because 1000 is not really a number of nodes we support in OCM)
```suggestion
|`--autoscaler-max-nodes-total 180`
```

Reply:
@osherdp what's the maximum supported value? We should limit it also in the UI.

Reply:
see https://issues.redhat.com/browse/OCM-3535
although not sure it's too critical to add a limit (just not document / default to something that is a bit too high)

Reply:
there's no validation for the maximal value of it
I think in general we shouldn't overthink it. We can always add a restriction in the future. We just need the docs / default values to be reasonable